imjoy/worker.py

```diff
@@ -93,11 +93,12 @@ def handle_method(conn, job, logger):
                 # args.append({'id': conn.id})
                 result = method(*args)
                 resolve(result)
-            except Exception as exc:  # pylint: disable=broad-except
+            except Exception:  # pylint: disable=broad-except
                 logger.error(
-                    "error in method %s: %s", job["name"], traceback.format_exc()
+                    "error in method %s: %s", job["name"], traceback.format_exc(
+                    )
                 )
-                reject(exc)
+                reject(traceback.format_exc())
```

Comment:
Sorry, I forgot, `traceback.format_exc()` will give us a string, we need to wrap it as an `Exception`, I think `reject(Exception(traceback.format_exc()))` should work. There maybe better way to do this, for the moment this would be easiest way to show the traceback in the ImJoy console.

For the other 3 places as well.

Reply:
Ok.